### ISSUE TITLE
Visual indication of when a course can / can't be unsettled

### ISSUE THREAD

**Issue by bnehoran** [TigerPathApp/tigerpath#92]
There should be some indication in the requirements tree of which settled courses can be unsettled and which cannot.

It might be annoying trying desperately to unsettle an automatically settled course and wondering why it just won't work, only to then realize that it just cannot be unsettled.

**Comment by churichard** [comment]
This could potentially be done by making an automatically settled course a little bit darker than a user settled course (this color is #99b1ff).
<img width="244" alt="image 2018-05-14 at 12 10 32 pm" src="https://user-images.githubusercontent.com/4218237/40009405-e48d961e-576f-11e8-8266-1a2549bf03b3.png">

**Comment by bnehoran** [comment]
@churichard yeah I think that might help for someone who knows that that's what's going on, but it might not be immediately clear to some users.
Maybe there's some way to make it more intuitively clear without needing to be explained.

**Comment by bnehoran** [comment]
Maybe the cursor should change as @churichard mentioned in #107 

Also, maybe mouseover text (the subtle text that often appears when you hover over something) saying things like
`This course was automatically assigned.`
or
`Click to assign the course to this requirement.`
or
`Click to unassign this course.`
would be useful.
